python/lsst/ap/association/skyBotEphemerisQuery.py

```diff
@@ -167,8 +167,7 @@ def run(self, visitInfos, visit):
         """
         # Grab the visitInfo from the raw to get the information needed on the
         # full visit.
-        visitInfo = visitInfos[0].get(
-            datasetType=self.config.connections.visitInfos)
+        visitInfo = visitInfos[0].get()
```

Comment:
I'm a bit confused by this `run` method since it takes `list[VisitInfo]` but then only uses the first one. If it's only using the first one why is it deferring the get?

Reply:
Ah. Okay. So the graph builder asks for visits and so this can be given exposures from a multi-snap visit and since the boresight is the same for both snaps it knows it only needs to use the first one.

Reply:
Deferral was part of the initial commit on d8a3fe4a3635e046502243477b705f79551932d6. You'll have to ask the authors where that came from.

Reply:
@eggls6 do you have a comment on the deferred loading? It doesn't matter for this PR but I am intrigued since it complicates the `run()` method to make it take a deferred handle and not a `VisitInfo`.